pkg/helm/controller/reconcile.go

```diff
@@ -248,7 +248,7 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 			status.SetCondition(types.HelmAppCondition{
```

Comment:
Hi @estroz, 

What we are changing here is the status name which will appear in the CR.status  to be aligned with helm ( part of the "new" helm-operator ). I agree that users might be checking due via a script, for example, the status for tests and etc and then, it would be a breaking change.  

My idea/suggestion is we do it before release 1.0.0 image for users are able to move smoothly. 

I will update the fragment. @joelanford wdyt?